.babelrc

```diff
@@ -0,0 +1,9 @@
+{
+  "presets": [
+    ["env", {
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7", "ie >= 9"]
```

Comment:
For project-wide setting,  I think we should use [browserslist](https://github.com/ai/browserslist) and put config in **package.json**, and tools support browserslist could be under the same config.

```js
{
  // package.json
  "browserslist": [
    "last 2 versions", 
    "safari >= 7", 
    "ie >= 9"
  ]
}
```